### DISCUSSION TITLE
Volume plugin

### DISCUSSION THREAD

**Discussion by Labrat-alter** [stokie-ant/lcdsmartie-laz#30] [Q&A] [answered]
Hi all, 
Is anyone have somewhere Volume.dll plugin? I'm trying to find it but with no success :/

**Comment by limbo666** [comment]
Unfrotunatelly volume plugin doesn't work anymore with modern LCD Smartie (it was too old for this) so I think there is no reasonn to provide you a non working file. 
The same may be true for volume7 which in my system is forcinng LCD Smartie to crash https://lcdsmartie.org/forums/viewtopic.php?f=8&t=2104&sid=f85efd5336d08393f20a9a90580f51b2

**BUT DON'T WORRY** 😉 I just worked on a new replacement plugin: That's **_vol.dll_**
⏬Download it here: [vol.zip](https://github.com/user-attachments/files/16738091/vol.zip)  

![image](https://github.com/user-attachments/assets/bb88dfb9-d477-40ee-8fe4-9aeec9c0fa4d)

Please let me know if this is what you need.
👋

**Comment by Labrat-alter** [comment]
Well it is but, it doesn't work 😞 
![obraz](https://github.com/user-attachments/assets/6f00989c-2c82-4ac3-a96a-f3dd7135862d)

**Reply by limbo666** [reply, marked as answer]
Try this: [vol.zip](https://github.com/user-attachments/files/16739478/vol.zip)
I bumped the .net framework requirement to 4.8  
If you don't have this framework isnatlled please download it from here: https://dotnet.microsoft.com/en-us/download/dotnet-framework/thank-you/net48-offline-installer)

**Comment by Labrat-alter** [comment]
same :/
![obraz](https://github.com/user-attachments/assets/8cd06779-2367-4425-b6cb-19d6aca9fc6d)

**Comment by limbo666** [comment]
That's strange.

Can you please replace LegacyLoader.exe and DNbrdge.dll  with the followiing? (backup yours first)
[LegacyLoader-DNBridge.zip](https://github.com/user-attachments/files/16740219/LegacyLoader-DNBridge.zip)

**Comment by Labrat-alter** [comment]
I've just replaced those files and no different

**Reply by limbo666** [reply]
This is becoming more complicated than it should be. I believe some runtime components might be missing from your system.

Please try the following steps:
[LCDSmartie_x64_Test.zip](https://github.com/user-attachments/files/16740453/LCDSmartie_x64_Test.zip)

- Extract the package and run LCD Smartie.
- Check the two screens that are enabled (Screen 1 & Screen 2).
- Do you see any messages coming from the DLLs?

It should work unless some runtime is missing from your system. If you still encounter errors, we will need to investigate further.

**Reply by limbo666** [reply]
Extract and run from the extracted folder. Do not replace your files. This is a complete LCD Smartie folder.

Additionaly run on powershell the following commands to verify that both 32bit and 64bit editions of .net framework are iinstalled on your system.

`Get-ChildItem 'HKLM:\SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full' | Get-ItemPropertyValue -Name Version`

`Get-ChildItem 'HKLM:\SOFTWARE\Wow6432Node\Microsoft\NET Framework Setup\NDP\v4\Full' | Get-ItemPropertyValue -Name Version`

This should give you a message like this:
![image](https://github.com/user-attachments/assets/8eb2cc64-5048-4574-8fb3-b45912a5a54a)

**Comment by limbo666** [comment]
🔥 **Update** 
Try this plugin version: [vol.zip](https://github.com/user-attachments/files/16744858/vol.zip) 
I have recompiled it to x64 
I think I accidentaly posted the 32bit edition of the compiled dll previously and this may caused the problem. 😳 
Please let me know if this gives the right result.

**Comment by Labrat-alter** [comment]
After long, long probably too long troubleshooting... It works
![obraz](https://github.com/user-attachments/assets/23d0d36b-5d40-427a-b880-916633595e36)
 Thanks for Your help 😄

**Comment by limbo666** [comment]
🔥
The latest version of this plugin can be found on: 
 https://github.com/Limbos-goodies-for-LCDSmartie
